src/routes/DictionaryScreen.tsx

```diff
@@ -33,13 +34,23 @@ const Header = styled.View`
 `
 
 interface Props {
-  navigation: StackNavigationProp<RoutesParams, 'DictionaryOverview'>
+  navigation: StackNavigationProp<RoutesParams, 'Dictionary'>
 }
 
 const DictionaryScreen = ({ navigation }: Props): ReactElement => {
   const documents = useLoadAllDocuments()
   const [searchString, setSearchString] = useState<string>('')
-  const filteredDocuments = documents.data?.filter(item => item.word.includes(searchString))
+  const searchStringWithoutArticle = ARTICLES.map(article => article.value).includes(
+    searchString.split(' ')[0].toLowerCase()
+  )
+    ? searchString.substring(searchString.indexOf(' ') + 1)
+    : searchString
+  const filteredDocuments = documents.data?.filter(item =>
+    item.word.toLowerCase().includes(searchStringWithoutArticle.toLowerCase().trim())
```

Comment:
Did you forget the normalization or is this on purpose? I think searching for e.g `Fon` should return `der Fön`.

Reply:
Do not really see the need, but added it.